packages/babel-template/test/index.js

```diff
@@ -219,7 +231,7 @@ describe("@babel/template", function() {
     });
   });
 
-  describe.only(".syntacticPlaceholders", () => {
+  describe(".syntacticPlaceholders", () => {
```

Comment:
😬

Reply:
We should probably lint for this 😬

Reply:
Actually, it looks like we should be linting for this already: https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/no-focused-tests.md is a recommended rule.

Reply:
We added it yesterday :stuck_out_tongue: 
https://github.com/babel/babel/pull/11119